app-creator/src/widgets/ui-map/ui-map.ts

```diff
@@ -0,0 +1,48 @@
+import {} from 'googlemaps';
+import { customElement, html, LitElement } from 'lit-element';
+
+@customElement('ui-map')
+export class Map extends LitElement {
+  /**
+   * Sets zoom property on map.
+   */
+  zoom = 4;
+
+  /**
+   * Sets center location on initial render.
+   */
+  center = { lng: -100.825, lat: 39.930546488601294 };
+
+  /**
+   * Sets map element.
+   */
+  map: google.maps.Map | null = null;
+
+  // Disable the shadow root, since it interferes with Google Maps.
+  // https://lit-element.polymer-project.org/guide/templates#renderroot
```

Comment:
This comment belongs to `createRenderRoot()` I think?

Reply:
Oops, there's a good chance that if Mohamed copied it from my ui-map implementation that the error is mine :P.